docs/md/dev-process.md

```diff
@@ -1,26 +1,22 @@
 # Development Process
 
-This is a development process
-we are tying to standardize
-within the team,
-and encourage ourselves
-to follow in most cases.
+This is a development process we are tying to standardize within the team and
+encourage ourselves to follow in most cases.
 
 ## The Swagger page
 
-WFL ships with a swaggger UI that documents all of the
-available endpoints. It's available at path `/swagger`, e.g.
+WFL ships with a swaggger UI that documents all of the available endpoints.
```

Comment:
Good time to remove the third g in swagger :)

Reply:
Ha! Good spot!